src/documentation/write_ntfs_fares.md

```diff
@@ -0,0 +1,36 @@
+# Writing NTFS fares
+## Introduction
+This document describes how fares specified in Navitia Transit Model are transformed into a [NTFS fare feed](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fare_extension_fr.md).
+
+In this initial version: 
+- only the files [prices.csv](#pricescsv) and [od_fares.csv](#od_farescsv) are written, as the NTM fares model does not yet support special conditions for the tickets
```

Comment:
Suggestion: To be verified but I think od_fares.csv is fully included in the model of fares.csv .
Can it be relevant to just keep the last one? Not fan of having multiple ways to express the same thing in a model, so I'd choose the larger part and progressively depreciate the restricted (yet simpler) od_fares.csv .

Reply:
I'd rather leave it as is for now, we need to keep read/write as simple as possible given the specificities of the input data (netex_nl). I note the remark though, as the internal model gets richer, the writing will be updated as well.